### PR TITLE
Add loading page

### PR DESCRIPTION
**Pull request by illseption**
- Added loading page to main website
- Added jQuery code to base.html for running loader animation
- Added loader animation class to base.css
Fixes #79

### CODE REVIEW

File: staticFiles/home/base.css [hunk 1]
```diff
@@ -4,4 +4,71 @@
   bottom: 0;
   left: 0;
   background-color: #222;
-}
\ No newline at end of file
+}
```
Comment (divyam3897) [inline staticFiles/home/base.css]: Remove this change.

File: templates/base.html [hunk 2]
```diff
@@ -20,9 +20,26 @@
...
+        jQuery(".spinner-wrapper").fadeOut(500);
+    });
+	</script>
 
   </head>
   <body  {% block body_style %} {% endblock %}>
+<!--CSS Spinner -->
+<div class="spinner-wrapper">
+  <div class="spinner">
+  	<div class="rect1"></div>
```
Comment (divyam3897) [inline templates/base.html]: give them better names to signify what rect they are.

Comment (illseption) [reply templates/base.html]: Should I make a new PR?

Comment (illseption) [reply templates/base.html]: If not, how do I edit this PR to show the changes?

File: staticFiles/home/base.css (hunk 1 above)
Comment (illseption) [reply staticFiles/home/base.css]: I didn't make this change in the original base.css file but its showing up on github for some reason :/

**Comment by illseption** [comment]
Made some changes I think. Please review.

**Comment by divyam3897** [comment]
@illseption I see no changes.

**Comment by illseption** [comment]
Made some changes. Overview: 
- Changed loader time from 500 to 1200
- Changed .rect names

File: staticFiles/home/base.css (hunk 1 above)
Comment (divyam3897) [reply staticFiles/home/base.css]: It's still to be removed.

**Comment by illseption** [comment]
Changes:
- Changed loader time from 1200 to 2500
- Squashed commits

**Comment by divyam3897** [comment]
It has been merged, thank you.

**Pull request merged**